dev-requirements.txt

```diff
@@ -15,3 +15,6 @@ gcp-devrel-py-tools==0.0.15
 
 # https://github.com/GoogleCloudPlatform/python-repo-tools/issues/23
 pylint<2.0;python_version<="2.7"
+
+pytz==2020.1
+tzlocal==2.1
```

Comment:
Never heard of `tzlocal`, could we use `python-dateutil` `tz.gettz()` instead if we're only using "get local timezone" functionality?

Reply:
Done, using the current time to do that.